modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+declare(strict_types = 1);
+
+use Drupal\Tests\oe_showcase\ExistingSite\ShowcaseExistingSiteTestBase;
+
+use Behat\Mink\Element\NodeElement;
+use Drupal\node\Entity\Node;
+use Drupal\node\NodeInterface;
+use Drupal\Tests\search_api\Functional\ExampleContentTrait;
+use Drupal\user\Entity\Role;
+
+/**
+ * Class to test List pages content type on existing site tests.
+ */
```

Comment:
The comment text is weird.
```suggestion
/**
 * Tests list pages.
 */
```